es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,55 @@
+import createElement from '../lib/create-element';
+import noop from "../lib/noop";
```

Comment:
After removing `default` from 'lib/index.js` you take advantage of that file and import both of these on 1 line:

```JAVASCRIPT
import { createElement, noop } from '../lib';
```